src/main/java/techItEasy/techItEasy/Controllers/ExceptionController.java

```diff
@@ -0,0 +1,4 @@
+package techItEasy.techItEasy.Controllers;
```

Comment:
Mooi dat je deze alvast hebt aangemaakt. Je kan deze in een volgende ronde vullen. Je kan ze gewoon ergens vandaan kopieren, bijv de uitwerkingen :)
@ExceptionHandler(value = RecordNotFoundException.class)
    public ResponseEntity<Object> exception(RecordNotFoundException exception) {

        return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);

    }